### DISCUSSION TITLE
11/14 (木) 20:00~ 全体ミーティング #4

### DISCUSSION THREAD

**Discussion by penguin2** [42-pong/42-pong#56] [Meetings]
- 進捗共有
- 課題項目をチームで確認
- 次回のタスク

**Comment by habvi** [comment]
### BE 進捗
- review
  - [x] team 単位でレビュワー指定
  - [x] team 外のチームから 1 人を自動指定
- backend
  - [x] workflow : formatter をかけずにエラー通知
  - [x] requirements.in でのパッケージの依存関係管理方法を楽に
  - [ ] Django 内で `.env` を取り込んで環境変数を使用 (superuser, SECRET_KEY)
  - [x] コンテナ起動時の healthcheck 追加
  - [ ] unit test workflow のディレクトリ指定に `paths` を使用

### 決めたいこと
- 課題 module について
- `/health GET` の次の機能実装内容について

**Comment by penguin2** [comment]
### 共有
  -  設定：レビュワー：チーム選択
  - 

### FE  共有、相談
- vanillaJS: TypeScript つき？
- tools
  - ビルドツール: [vite](https://ja.vite.dev/)
  - リンター、フォーマッター: [biome](https://biomejs.dev/ja/)
  - テストツール: [vitest](https://vitest.dev/) (mock test も?)
  - docker で nginx ? (受付として別のコンテナ？nginx などのサーバーを持つ？)
-

**Comment by habvi** [comment]
- (FE / BE) server用コンテナ要相談 -> 後からで余裕があれば独立させる？適宜。

### pong 開発順序
まずデモ的に簡単にローカルで動くゲームを作り、その後で module 含め仕様を決めて拡張していく
- user
- game
  - 2 人対戦
  - トーナメント 




### 後から追加できればしたい興味のあること
- Gameplay & UX : Live chat
- Graphics : 3D
- User Management : Implementing a remote authentication
- AI : Introduce an AI Opponent
- websocket 使ってみたい

### 余裕があれば
- Matchmaking
